swift/downgrades/04ef9ecaa9e06a587f252f147462861e3d32846a/upgrade.properties

```diff
@@ -0,0 +1,14 @@
+description: Replacing types and expressions related to parameter packs
```

Comment:
Not sure I understand this description. Usually I just go with "Revert ***" (for example, "Revert adding parameter pack extraction")

Reply:
Here you go https://github.com/github/codeql/pull/14750